betovarg/lucysanchez
# LS-5: illustrations

## Steps to test

- go the the repository branch `feature/LS-5-illustrations`.
- clean caches of Grav on admin Grav --> bin/grav clear-cache.
- go to localhost:8080/illustrations.
- review if the Gallery appears and have the ligthbox function.
- also check if navigation on lightbox is enabled.

NOTE: navigation when lightbox is active still in progress
